community/cassandra-cpp-driver/APKBUILD

```diff
@@ -3,7 +3,7 @@
 pkgname=cassandra-cpp-driver
 _pkgname=cpp-driver
 pkgver=2.10.0
-pkgrel=0
+pkgrel=1
```

Comment:
no need to bump as no changes are made

Reply:
Thanks, amended PR to remove bump of pkgrel